plugins/evernote.koplugin/main.lua

```diff
@@ -507,4 +667,30 @@ function EvernoteExporter:exportBooknotesToTXT(title, booknotes)
     end
 end
 
+function EvernoteExporter:exportBooknotesToJoplin(client, title, booknotes)
+    if not client:ping() then
+        error("Cannot reach Joplin server")
+    end
+
+    local note_guid = client:findNoteByTitle(title, self.joplin_notebook_guid)
+    local note = ""
+    for _, chapter in ipairs(booknotes) do
+        if chapter.title then
+            note = note .. "\n\t*" .. chapter.title .. "*\n\n * * *"
+        end
+
+        for _, clipping in ipairs(chapter) do
+            ---@todo Do something to prettify epoch time, write it.
```

Comment:
```suggestion
            --- @todo Do something to prettify epoch time, write it.
```
However, this strikes me as an odd todo. It takes seconds to write os.date("%Y-%m-%d", epoch_time). ;-)